documentation/pe_puppetdb_notes.markdown

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
PE specific notes, more like a directory of PE features/differences for users to use for navigation, but yeah I imagine this will end up mostly looking like release notes.